core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -178,7 +182,6 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
     super.preStart()
 
     consumer = settings.createKafkaConsumer()
-    currentPollTask = schedulePollTask()
```

Comment:
ok, so it's deferring the first poll until after the first Subscribe (or similar) message